packages/mail/src/classes/mail-service.js

```diff
@@ -112,7 +112,7 @@ class MailService {
 
       //Pass to callback if provided
       if (cb) {
-        cb(error, null);
+        return cb(error, null);
```

Comment:
The promise will never be rejected (thus won't return a `Promise`) if you stop the code here, so any code that uses this function as a promise and provides a callback will crash the program.

Reply:
Good pickup, thanks!

I've reverted the change and added an `eslint-disable-next-line` rather than restructure the code.